### ISSUE TITLE
I used a2d2 dataset, the input image size is 1208 X 1920. Should I modify the image size to the same size as coco dataset before feed into the model for fine turning/

### ISSUE THREAD

**Issue by metacav** [xingyizhou/CenterNet#972]
I used a2d2 dataset, the input image size is 1208 X 1920. Should I modify the image size to the same size as coco dataset before feed into the model for fine turning? Can main.py convert both image and annotation to the target size of 512X512?

**Comment by Dipankar1997161** [comment]
yes, it is recommended to do that, or if you are ok with 512x512, then it shouldn't be a problem

**Comment by Fortuneteller6** [comment]
What is best size when u input the image？Is the default 1208*1920 best or Using the 512*512 best？
